.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Don't commit this.

Reply:
So the current .gitignore file is explicitly allowing this file:

```
### VisualStudioCode ###
.vscode/*
!.vscode/settings.json
!.vscode/tasks.json
!.vscode/launch.json
!.vscode/extensions.json
.history
```